app_dart/lib/src/request_handling/static_file_handler.dart

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io' show ContentType, HttpResponse;
+import 'dart:typed_data';
+
+import 'package:cocoon_service/cocoon_service.dart';
+import 'package:file/file.dart';
+import 'package:file/local.dart';
+import 'package:meta/meta.dart';
+import 'package:mime/mime.dart';
+
+import 'body.dart';
+import 'exceptions.dart';
+
+/// A class based on [RequestHandler] for serving static files.
+@immutable
+class StaticFileHandler<T extends Body> extends RequestHandler<Body> {
+  /// Creates a new [StaticFileHandler].
+  const StaticFileHandler(this.filePath,
+      {@required Config config, FileSystem fs})
```

Comment:
You can use a default value here -- `this.fs = const LocalFileSystem()` (and no harm in making the instance variable public).

Reply:
Added, thank you!